msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -925,7 +927,19 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
 
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
-|`contexts`| Array | 4 | | Specifies the Microsoft 365 application window in which the ribbon customization is available to the user. Each item in the array is a member of a string array. <br>Supported values: `mailRead`, `mailCompose`, `meetingDetailsOrganizer`, `meetingDetailsAttendee`|
+|`contexts`| Array | 4 | | Specifies the Microsoft 365 application window in which the ribbon customization is available to the user. Each item in the array is a member of a string array. <br>Supported values: `mailRead`, `mailCompose`, `meetingDetailsOrganizer`, `meetingDetailsAttendee` , `onlineMeetingDetailsOrganizer`, `logEventMeetingDetailsAttendee`, `default`|
+|`customMobile`| Array | 10 | | Defines mobile group items. |
+|`customMobile.id`| String | |✔️| Specify the Id of the group. Used for mobileMessageRead ext point.|
+|`customMobile.label`| String | |✔️| Short label of the control. Maximum length is 32 characters.|
+|`customMobile.controls`| Array | |✔️| Defines mobileControl group items. |
+|`customMobile.controls.id`| String | |✔️| Specify the Id of the button like msgReadFunctionButton. |
+|`customMobile.controls.type`| String | |✔️| mobileButton |
+|`customMobile.controls.label`| String | |✔️| Short label of the control. Maximum length is 32 characters. |
+|`customMobile.controls.icons`| Array | |✔️| Custom mobile icon. |
+|`customMobile.controls.icons.size`| Number | |✔️| Size in pixels of the icon. Three image sizes are required (25, 32, and 48 pixels). |
+|`customMobile.controls.icons.url`| Number | |✔️| Url to mobile icon|
+|`customMobile.controls.icons.scale`| Number | |✔️| How to scale - 1,2,3 for each image. This attribute specifies the UIScreen.scale property for iOS devices. |
+|`customMobile.controls.actionId`| String | |✔️| The ID of an action defined in runtimes. Maximum length is 64 characters. |
```

Comment:
```suggestion
|`customMobile.id`| String | 250 |✔️| Specify the Id of the group. Used for mobileMessageRead ext point.|
|`customMobile.label`| String | 32 characters|✔️| Short label of the control.|
|`customMobile.controls`| Array | 10 |✔️| Defines mobileControl group items. |
|`customMobile.controls.id`| String | 250|✔️| Specify the Id of the button like msgReadFunctionButton. |
|`customMobile.controls.type`| String | |✔️| mobileButton |
|`customMobile.controls.label`| String | 32 characters |✔️| Short label of the control. |
|`customMobile.controls.icons`| Array | |✔️| Custom mobile icon. |
|`customMobile.controls.icons.size`| Number | |✔️| Size in pixels of the icon. Three image sizes are required (25, 32, and 48 pixels). |
|`customMobile.controls.icons.url`| Number | |✔️| Url to mobile icon|
|`customMobile.controls.icons.scale`| Number | |✔️| How to scale - 1,2,3 for each image. This attribute specifies the UIScreen.scale property for iOS devices. |
|`customMobile.controls.actionId`| String | 64 characters |✔️| The ID of an action defined in runtimes. |
```

Reply:
@mobajemu-msft , Can you also please check `customMobile.controls.icons`, in the Dev Preview manifest it says Minimun and Max have the same number, do we need to update the Maximum size filed here?

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/8a8d4f49-0430-45a1-9d33-b978ae354d3d)